.gitattributes

```diff
@@ -0,0 +1,304 @@
+*.ini       text
+*.plist     text
+*.toml      text
+*.xml       text
+*.yml       text
+*.yaml      text
+
+# Plain text data
+*.cdl       text
+*.csv       text
+*.dif       text
+*.geojson   text
+*.gml       text
+*.kml       text
+*.sql       text
+*.tab       text
+*.tsv       text
+*.wkt       text
+
+# Other text files
+*.diff      -text
+*.patch     -text
+
+# Special files
+.*rc            text
+.checkignore    text
+.ciocheck       text
+.ciocopyright   text
+.editorconfig   text
+.gitattributes  export-ignore
+.gitconfig      export-ignore
+.gitignore      export-ignore
+.gitmodules     export-ignore
+.gitkeep        export-ignore
+*.lektorproject text
+.nojekyll       text
+.project        text
+
+AUTHORS         text
+CHANGELOG       text
+CHANGES         text
+CONTRIBUTING    text
+INSTALL         text
+license         text
+LICENSE         text
+NEWS            text
+NOTICES         text
+readme          text
+*README*        text
+RELEASE         text
+TODO            text
+
+browserslist    text
+contents.lr     text
+makefile        text
+Makefile        text
+MANIFEST.in     text
+
+
+# Declare files that will always have CRLF line endings on checkout.
+*.bat           text eol=crlf
+*.cmd           text eol=crlf
+*.vbs           text eol=crlf
+*.vb            text eol=crlf
+
+
+# Denote all files that are truly binary and should not be modified.
+
+# Executable
+*.app       binary
+*.bin       binary
+*.deb       binary
+*.dll       binary
+*.dylib     binary
+*.elf       binary
+*.exe       binary
+*.ko        binary
+*.lib       binary
+*.msi       binary
+*.o         binary
+*.obj       binary
+*.pyc       binary
+*.pyd       binary
+*.pyo       binary
+*.rdb       binary
+*.Rdb       binary
+*.rdx       binary
+*.Rdx       binary
+*.rpm       binary
+*.so        binary
+*.sys       binary
+
+# Data
+*.cdf       binary
+*.db        binary
+*.dta       binary
+*.feather   binary
+*.fit       binary
+*.fits      binary
+*.fts       binary
+*.fods      binary
+*.geotiff   binary
+*.gpkg      binary
+*.h4        binary
+*.h5        binary
+*.hdf       binary
+*.hdf4      binary
+*.hdf5      binary
+*.mat       binary
+*.nc        binary
+*.npy       binary
+*.npz       binary
+*.odb       binary
+*.ods       binary
+*.p         binary
+*.parquet   binary
+*.pickle    binary
+*.pkl       binary
+*.rdata     binary
+*.Rdata     binary
+*.RData     binary
+*.rda       binary
+*.Rda       binary
+*.rds       binary
+*.Rds       binary
+*.sav       binary
+*.sqlite    binary
+*.wkb       binary
+*.xls       binary
+*.XLS       binary
+*.xlsx      binary
+*.XLSX      binary
+
+# Documents
+*.doc       binary
+*.DOC       binary
+*.docx      binary
+*.DOCX      binary
+*.epub      binary
+*.fodp      binary
+*.fodt      binary
+*.odp       binary
+*.odt       binary
+*.pdf       binary
+*.PDF       binary
+*.ppt       binary
+*.PPT       binary
+*.pptx      binary
+*.PPTX      binary
+*.rtf       binary
+*.RTF       binary
```

Comment:
Is this `Documents` section necessary here?

Reply:
See [my comment](https://github.com/spyder-ide/qtpy/pull/278#issuecomment-963716528)